homework_1/decorators/cache_decorator.py

```diff
@@ -48,10 +48,3 @@ def wrapper(*args, **kwargs):
         return wrapper
```

Comment:
Remove for statement. You dont need it.  Extend wrapper._cache to store count of calls.

Reply:
I guess you meant that I need to delete the "return cache_data". If I'll remove the "return wrapper" function will collapse.
wrapper._cache is already extended and already stores the number of calls. Now there is a nested dictionary, but I will redo it by the list in the dictionary.

Reply:
I guess I did what you are talking about